src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/DEC2HEX.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Functions.Delegation;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // DEC2HEX(number:n, [places:n])
+    internal sealed class DEC2HEXFunction : BuiltinFunction
```

Comment:
>DEC2HEX [](http://example.com/codeflow?start=26&length=7)

While Excel function names are all CAPITALIZED, we use title casing in Power Fx, so the function name should be Dec2Hex (similarly with the file names)